Rethinking/end-of-chapter-practice-problems/ch-2.ipynb

```diff
@@ -0,0 +1,752 @@
+{
```

Comment:
<p>Small typo: four of them *are* consistent with the the event "other side is also black."</p>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>